### PR TITLE
[yeelight] Add sptrip6 device

### PR DESCRIPTION
**Pull request by goopilot**
Signed-off-by: Alexandr Salamatov [goopilot@gmail.com](mailto:goopilot@gmail.com)

Added support of a new LED strip light (strip6). This issue was mentioned https://community.openhab.org/t/yeelight-new-stripe-model/118764

It looks very much like original stripe and has the following capabilities:

strip6
{ "ip": "X.X.X.X", "port": 55443, "capabilities": { "id": "0x00000000175f6828", "model": "strip6", "fw_ver": "20", "support": "get_prop set_default set_power toggle set_bright set_scene cron_add cron_get cron_del start_cf stop_cf set_name set_adjust adjust_bright set_ct_abx adjust_ct adjust_color set_rgb set_hsv set_music udp_sess_new udp_sess_keep_alive udp_chroma_sess_new", "power": "on", "bright": "100", "color_mode": "1", "ct": "3524", "rgb": "7864244", "hue": "147", "sat": "53", "name": "" } }

stripe
{ "ip": "X.X.X.X", "port": 55443, "capabilities": { "id": "0x000000000361cae2", "model": "stripe", "fw_ver": "50", "support": "get_prop set_default set_power toggle set_bright start_cf stop_cf set_scene cron_add cron_get cron_del set_ct_abx set_rgb set_hsv set_adjust adjust_bright adjust_ct adjust_color set_music set_name", "power": "on", "bright": "100", "color_mode": "1", "ct": "6500", "rgb": "16777215", "hue": "0", "sat": "0", "name": "" } }

So new supported commands are: udp_sess_new, udp_sess_keep_alive, udp_chroma_sess_new which I did not implement.

https://openhab.jfrog.io/artifactory/libs-pullrequest-local/org/openhab/addons/bundles/org.openhab.binding.yeelight/4.0.0-SNAPSHOT/org.openhab.binding.yeelight-4.0.0-SNAPSHOT.jar

### CODE REVIEW

**Comment by goopilot** [comment]
[jlaur](https://github.com/jlaur) crated pull request agains main branch as requested.

File: bundles/org.openhab.binding.yeelight/src/main/resources/OH-INF/config/config.xml [hunk 1]
```diff
@@ -15,7 +15,7 @@
 			<description>Duration of transition of events such as on/off, change of brightness and change of color, in
 				milliseconds.</description>
 			<unitLabel>milliseconds</unitLabel>
-			<default>500</default>
+			<default>5000</default>
```
Comment (jlaur) [inline bundles/org.openhab.binding.yeelight/src/main/resources/OH-INF/config/config.xml:18]: Why this change?

**Review by jlaur — CHANGES_REQUESTED** [review]
Thanks! I see some preparation was already done in #10222.

File: bundles/org.openhab.binding.yeelight/src/main/resources/OH-INF/config/config.xml (hunk 1 above)
Comment (goopilot) [reply bundles/org.openhab.binding.yeelight/src/main/resources/OH-INF/config/config.xml:18]: There is an issue that device stays offline with default duration 500. I experienced it myself and was able to resolve by increasing duration to 5000.
Also same issue happens to others: https://community.openhab.org/t/yeelight-color-led-bulb-offline-using-yeelight-binding/96583/4

Comment (jlaur) [reply bundles/org.openhab.binding.yeelight/src/main/resources/OH-INF/config/config.xml:18]: So this is not transition time, i.e. the time it takes to from light being on to light being fully off, for example? In that case 5 seconds seems awfully slow as default value, but I may of course have read the description and documentation wrong. Transition time put into a request should not cause those problems mentioned?

Please note that default value is also here:
https://github.com/openhab/openhab-addons/blob/b395d0e2271077409f15c9f7ac17ba3814d064f9/bundles/org.openhab.binding.yeelight/src/main/java/org/openhab/binding/yeelight/internal/handler/YeelightHandlerBase.java#L351-L353

Comment (goopilot) [reply bundles/org.openhab.binding.yeelight/src/main/resources/OH-INF/config/config.xml:18]: Yeah, this value also affects how much is a wait time between sending every new dice command, so setting it to 5s makes device less responsive. There is an option to keep default as 500 but update comment so users will see that this value affects device online status:

https://github.com/openhab/openhab-addons/blob/b395d0e2271077409f15c9f7ac17ba3814d064f9/bundles/org.openhab.binding.yeelight/src/main/resources/OH-INF/i18n/yeelight.properties#L30

`Duration of transition of events such as on/off, change of brightness and change of color, in milliseconds. Try to increase if device is going offline.`

Comment (jlaur) [reply bundles/org.openhab.binding.yeelight/src/main/resources/OH-INF/config/config.xml:18]: In context of this PR, in my opinion, we should not change this default.

Would it be possible to explain why transition time and offline problems are correlated, and suggest increasing the value as a work-around? This would be a nice improvement for the README. I'm not sure if adding this to the parameter description would give enough information, and might just raise more questions? It would be okay to improve the documentation in this PR, but you could also create a new dedicated PR to not block this one as it's not related.

Comment (goopilot) [reply bundles/org.openhab.binding.yeelight/src/main/resources/OH-INF/config/config.xml:18]: ok sounds good

**Review by jlaur — APPROVED** [review]
LGTM

File: bundles/org.openhab.binding.yeelight/src/main/resources/OH-INF/config/config.xml (hunk 1 above)
Comment (goopilot) [reply bundles/org.openhab.binding.yeelight/src/main/resources/OH-INF/config/config.xml]: @jlaur I opened an issue for device going offline https://github.com/openhab/openhab-addons/issues/14635
